docs-chef-io/content/inspec/resources/sshd_active_config.md

```diff
@@ -0,0 +1,93 @@
++++
+title = "sshd_active_config resource"
+draft = false
+gh_repo = "inspec"
+platform = "linux"
+[menu]
+  [menu.inspec]
+    title = "sshd_active_config"
+    identifier = "inspec/resources/os/sshd_active_config.md sshd_active_config resource"
+    parent = "inspec/resources/os"
++++
+Use the `sshd_active_config` Chef InSpec audit resource to test configuration data for the OpenSSH daemon located at `/etc/ssh/sshd_active_config` on Linux and Unix platforms. sshd---the OpenSSH daemon---listens on dedicated ports, starts a daemon for each incoming connection, and then handles encryption, authentication, key exchanges, command execution, and data exchanges.
```

Comment:
I don't think this path is what you meant, the active config finds the path of the sshd_config file by interrogating the active sshd process for the config file it is loaded with. Then we can say, all other functions and properties of the ssh and sshd resource are the same with the addition of the `active_path` variable available to the `sshd_active_config` resource which will be the path of the config file the sshd service reported as loaded.

Reply:
My mistake. This has been corrected.